functions/reportToIWF.ts

```diff
@@ -47,11 +47,25 @@ export type Event = {
   Reporter_Last_Name?: string;
   Reporter_Email_ID?: string;
   Reporter_Description?: string;
-  request: { cookies: {}; headers: {}; }
+  request: { cookies: {}; headers: {} };
 };
 
 export const handler: ServerlessFunctionSignature<EnvVars, Event> = TokenValidator(
-  async (context: Context<EnvVars>, event: Required<Pick<Event, 'Reported_URL' | 'Reporter_Anonymous' | 'Reporter_Description' | 'Reporter_Email_ID' | 'Reporter_First_Name' | 'Reporter_Last_Name'>>, callback: ServerlessCallback) => {
+  async (
+    context: Context<EnvVars>,
+    event: Required<
+      Pick<
+        Event,
+        | 'Reported_URL'
+        | 'Reporter_Anonymous'
+        | 'Reporter_Description'
+        | 'Reporter_Email_ID'
+        | 'Reporter_First_Name'
+        | 'Reporter_Last_Name'
+      >
+    >,
+    callback: ServerlessCallback,
+  ) => {
```

Comment:
This change does not seems correct. Why modify the `Event` type at all if we don't pass that information to the `handler`? `event: Event` might be the one you want here?